roles/ansible_service_broker/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 # do any asserts here
 
 - include: install.yml
-  when: ansible_service_broker_install | default(false) | bool
+  when: ansible_service_broker_install | default(true) | bool
```

Comment:
This will cause the deployment to error if the catalog is not set to deploy, because the broker resource will fail on creation.